brunch-config.js

```diff
@@ -53,5 +53,8 @@ exports.config = {
 
   npm: {
     enabled: true,
+    globals: {
+      Vue: 'vue/dist/vue.common.js'
```

Comment:
Nope, `import Vue` is still broken: https://github.com/vuejs/vue-router/issues/713#issuecomment-250966053